examples/editor_iframe.html

```diff
@@ -2,6 +2,10 @@
 <html lang="en">
 <head>
 <style>
+body {
+  line-height: 17px;
```

Comment:
the line height is important since if it is not explicitly set then when queried the result will simply be "normal", which assumes a certain multiplier which is browser specific and unknown, as far as I can tell.